### ISSUE TITLE
rustflags set for wasm-builder is actually not working

### ISSUE THREAD

**Issue by dtynn** [filecoin-project/ref-fvm#322]
rustflags set like [https://github.com/filecoin-project/ref-fvm/blob/master/actors/miner/build.rs#L6-L10](https://github.com/filecoin-project/ref-fvm/blob/master/actors/miner/build.rs#L6-L10) is actually not working (at least for  `1.60.0-nightly` or `nightly-2021-12-14` which l've already tested)

the latest [substrate-wasm-builder 5.0.0-dev in it's git repo](https://github.com/paritytech/substrate/tree/master/utils/wasm-builder) works fine.

and also, some flags are incorrect(or out of date?) like `-Coverflow-checks=true` and `-Clto=true`, all the `true`s should be switched to `yes`

**Comment by dtynn** [comment]
some error info captured:
```
  --- stderr
  error: failed to run `rustc` to learn about target-specific information

  Caused by:
    process didn't exit successfully: `rustc - --crate-name ___ --print=file-names -C link-arg=--export-table -C link-arg=--import-memory -Cinstrument-coverage -Zno-profiler-runtime -Ctarget-feature=+crt-static -Cpanic=abort -Coverflow-checks=true -Clto=true -Copt-level=z --target wasm32-unknown-unknown --crate-type bin --crate-type rlib --crate-type dylib --crate-type cdylib --crate-type staticlib --crate-type proc-macro --print=sysroot --print=cfg` (exit status: 1)
    --- stderr
    error: incorrect value `true` for codegen option `overflow-checks` - one of: `y`, `yes`, `on`, `n`, `no`, or `off` was expected
```

```
  --- stderr
  error: failed to run `rustc` to learn about target-specific information

  Caused by:
    process didn't exit successfully: `rustc - --crate-name ___ --print=file-names -C link-arg=--export-table -C link-arg=--import-memory -Cinstrument-coverage -Zno-profiler-runtime -Ctarget-feature=+crt-static -Cpanic=abort -Coverflow-checks=yes -Clto=true -Copt-level=z --target wasm32-unknown-unknown --crate-type bin --crate-type rlib --crate-type dylib --crate-type cdylib --crate-type staticlib --crate-type proc-macro --print=sysroot --print=cfg` (exit status: 1)
    --- stderr
    error: incorrect value `true` for codegen option `lto` - either a boolean (`yes`, `no`, `on`, `off`, etc), `thin`, `fat`, or omitted was expected
```

**Comment by Stebalien** [comment]
Nice catch! Fix in #323.
